README.md

```diff
@@ -64,10 +64,13 @@ $ sudo image-builder build qcow2 --distro centos-9
 this will create a directory `centos-9-qcow2-x86_64` under which the
 output is stored.
 
-With the `--extra-artifacts=manifest` an
+With the `--with-manifest` option an
 [osbuild](https://github.com/osbuild/osbuild) manifest will be
 placed in the output directory too.
 
+With the `--with-sbom` option an SPDX SBOM document will be
```

Comment:
There is a `### SBOMs` section in the README a little bit below that would also need this update

Reply:
Ah yea, I had just spotted that one as well; I've adjusted the last commit to also change it there.